community/aqua/chemistry/h2_iqpe.ipynb

```diff
@@ -48,7 +48,7 @@
     "]\n",
     "\n",
     "backends = [\n",
-    "    Aer.get_backend('qasm_simulator'),\n",
+    "    get_aer_backend('qasm_simulator'),\n",
```

Comment:
why are we wrapping up backends

Reply:
@jaygambetta , we created that wrapper because Aer is not on GitHub yet, and we cannot use it directly and be sure that the code we write now will work, so the wrapper implements a fallback.  We need that wrapper to test Aqua with Aer whenever Aer is installed, which is good because it crashed in one unit test.  However, we did not mean for that wrapper to be used in public tutorials.  We are going to use Legacy for now.